packages/ripple-nuxt-tide/modules/search/lib/searchmixin.js

```diff
@@ -80,6 +81,10 @@ const searchMixin = {
       } catch (e) {
         const msg = 'Search isn\'t working right now, please try again later.'
         this.errorMsg = msg
+
+        if (process.server) {
+          logger.error('Retrieving search results failed in getSearchResults()', { e, label: 'SearchMixin' })
```

Comment:
Hey @rubyllamadora Thanks for adding the log here. The error object need to be added as `{ error: e, label: 'SearchMixin'}` otherwise it won't be used.

Reply:
Nice pickup! I'll rename parameter to error so can just have `{error, label:...` - cleaner :P